src/recipes.jl

```diff
@@ -550,7 +550,7 @@ Plots.@deps stepbins path
 function _auto_binning_nbins{N}(vs::NTuple{N,AbstractVector}, dim::Integer; mode::Symbol = :auto)
     _cl(x) = max(ceil(Int, x), 1)
```

Comment:
So, on 0.6 we'll have to do `_cl(x) = ceil(Int, NaNMath.max(x, one(x)))` - that is the whole point of this PR :-D
I'll put in the change.